services/bonita-page/bonita-page-impl/src/main/resources/org/bonitasoft/engine/page/impl/hibernate/page.queries.hbm.xml

```diff
@@ -50,4 +50,10 @@
 		WHERE pagemapping.key = :key
 	</query>
 
+	<query name="getPageMappingByPageId">
```

Comment:
this query could be tested unitary 
see example in bonita-engine/bonita-integration-tests/bonita-query-tests/src/test/java/org/bonitasoft/engine/page/PageQueriesTest.java